packages/experiments/src/components/CollapsibleSection/CollapsibleSection.state.tsx

```diff
@@ -48,7 +48,16 @@ export const useCollapsibleSectionState: ICollapsibleSectionComponent['state'] =
 
   const _onRootKeyDown = useCallback((ev: React.KeyboardEvent<Element>) => {
     const rootKey = getRTL() ? KeyCodes.right : KeyCodes.left;
-    if (ev.which === rootKey && ev.target !== titleElementRef.current && titleElementRef.current) {
+
+    // If left/right keypress originates from text input or text area inside collapsible section,
+    // ignore the event.
+    if (
+      ev.which === rootKey &&
+      ev.target !== titleElementRef.current &&
+      titleElementRef.current &&
+      !(ev.target instanceof HTMLTextAreaElement) &&
+      !(ev.target instanceof HTMLInputElement && ev.target.type === 'text')
```

Comment:
I wonder if this will scale if there are other instances we find but I'm not seeing a better solution right now.

Reply:
We have similar code at https://github.com/OfficeDev/office-ui-fabric-react/blob/master/packages/office-ui-fabric-react/src/utilities/selection/SelectionZone.tsx#L389, I tightened the logic a bit (`type === 'text` for `<input>`) here.